internal/git/git.go

```diff
@@ -204,16 +204,12 @@ func (s *Service) LocateServiceRelease(ctx context.Context, r *git.Repository, e
 }
 
 func locateServiceReleaseCondition(env, service string) conditionFunc {
-	r := regexp.MustCompile(fmt.Sprintf(`(?i)\[%s/%s] release`, regexp.QuoteMeta(env), regexp.QuoteMeta(service)))
-	return func(commitMsg string) bool {
-		if env == "" {
-			return false
-		}
-		if service == "" {
-			return false
-		}
-		return r.MatchString(commitMsg)
+	if env == "" || service == "" {
+		return falseConditionFunc
 	}
+	return commitinfo.LocateRelease(func(c commitinfo.CommitInfo) bool {
+		return strings.ToLower(c.Service) == strings.ToLower(service) && strings.ToLower(c.Environment) == strings.ToLower(env)
```

Comment:
Same again with EqualFold

```suggestion
		return strings.EqualFold(c.Service, service) && strings.EqualFold(c.Environment, env)
```